docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+services:
+  cinema:
+    build:
+      context: .
+    env_file:
+      - .env
+    ports:
+      - "8000:8000"
+    command: >
+      sh -c "python manage.py wait_for_db &&
+            python manage.py migrate &&
+            python manage.py runserver 0.0.0.0:8000"
```

Comment:
The command sequence uses `&&` to chain commands. If any command fails, the subsequent commands will not run, which is generally desired. However, ensure that the `wait_for_db` command is robust enough to handle database connection retries effectively.